test/externalTests/perpetual-pools.sh

```diff
@@ -66,10 +66,9 @@ function perpetual_pools_test
     force_hardhat_unlimited_contract_size "$config_file" "$config_var"
     yarn install
 
-    # The project depends on @openzeppelin/hardhat-upgrades, which is currently not prepared
-    # for the parallel compilation introduced in Hardhat 2.9.0.
-    # TODO: Remove when https://github.com/OpenZeppelin/openzeppelin-upgrades/issues/528 is fixed.
-    yarn add hardhat@2.8.4
+    # With ethers.js 5.6.2 many tests for revert messages fail.
+    # TODO: Remove when https://github.com/ethers-io/ethers.js/discussions/2849 is resolved.
+    yarn add ethers@5.6.1
```

Comment:
An here it is totally ok to just switch from `hardhat` to `ethers.js`?

Reply:
These are two separate workarounds that just happen to be next to each other. The `hardhat` add command was a workaround for #12733 and is no longer needed.